src/components/symptom-shark/container/LogEntryEdit/LogEntryEdit.tsx

```diff
@@ -181,8 +181,8 @@ export default function (props: SymptomSharkLogEntryEditProps) {
     }
 
     return (
-        <>
-            <NavigationBar title={dateLabel} showBackButton={true} variant="compressed" />
+        <div className="mdhui-ss-log-entry-edit-wrapper">
```

Comment:
Would this ever go into the View builder?

Reply:
No, not currently planned to be incorporated into the view builder since it's a detail screen with parameters...so it's a bit awkward to put there at the moment